.gitmodules

```diff
@@ -10,7 +10,7 @@
 	url = git@github.com:BeTR-biogeochemistry-modeling/sbetr.git
 [submodule "components/mpas-source"]
 	path = components/mpas-source
-	url = git@github.com:MPAS-Dev/MPAS-Model.git
+	url = git@github.com:sbrus89/MPAS-Model.git
```

Comment:
is there a reason to point at your own fork here?

Reply:
This is just temporary. See below.